test/threads_exec.jl

```diff
@@ -1312,4 +1313,80 @@ end
         end
     end
 end
+
+@testset "task time counters" begin
+    @testset "enabled" begin
+        try
+            Base.task_timing(true)
+            start_time = time_ns()
+            t = Threads.@spawn peakflops()
+            wait(t)
+            end_time = time_ns()
+            wall_time_delta = end_time - start_time
+            @test t.is_timing_enabled
+            @test Base.task_cpu_time_ns(t) > 0
+            @test Base.task_wall_time_ns(t) > 0
+            @test Base.task_wall_time_ns(t) >= Base.task_cpu_time_ns(t)
+            @test wall_time_delta > Base.task_wall_time_ns(t)
```

Comment:
Can you add a tiny test that these aren't still ticking up? That they're fixed to a value once the task ends.

Reply:
added